models/userSchema.js

```diff
@@ -0,0 +1,70 @@
+const { checkSchema } = require('express-validator');
+const mongoose = require('mongoose');
+
+const userSchema = new mongoose.Schema(
+  checkSchema(
+    {
+      name: {
+        type: String,
+        required: true,
+        maxLength: 22,
+      },
+      email: {
+        type: String,
+        required: true,
+        unique: true,
+        trim: true,
+        lowercase: true,
+        maxLength: 20,
+        isEmail: {
+          bail: true,
+          errorMessage: 'Invalid Email',
+        },
+      },
+      phone: {
+        type: Number,
+        maxLength: 12,
+        trim: true,
+        isInt: {
+          errorMessage: 'Invalid Phone Number',
+          bail: true,
+        },
```

Comment:
make another schema for **callingCode**. Like +91 or +131. We need that as well with contact. Keep the name as **callingCode**